test/data/LocalDataSource.js

```diff
@@ -61,10 +61,10 @@ LocalSource.prototype = {
                 observer.onNext(output);
                 observer.onCompleted();
             }
-            if (wait > 0) {
-                setTimeout(exec, wait);
-            } else {
+            if (wait === false) {
```

Comment:
Agree with the decision to make "0" mean setTimeout(0). However would've used null/undefined to indicate no wait. That way the member could just be optional, and if not included, there would be no async.

Reply:
thanks! points from discussion with @jhusain :

it did already was no wait if you missed the property (because wait gets defaulted to `false` earlier), but that did leave out the case where wait is explicitly sent in as undefined. agreed that missing the property should behave the same as setting it to undefined, so now:
using `undefined` to indicate no wait. not caring about `null` or other possible types at this point (this is test helper code).